src/store/houses/actions.ts

```diff
@@ -2,43 +2,50 @@ import { ActionTree } from 'vuex';
 import { RootStateInterface } from '@/store/types';
 import { HousesStateInterface, HousesMutationsEnum, HousesActionsEnum, Actions } from '@/store/houses/types';
 import { HTTP } from '@/core/api/http-common';
+import { HouseModel } from './../../shared/models/house.model';
+import { HouseDTOModel } from './../../shared/models/houseDTO.model';
 
 export const actions: ActionTree<HousesStateInterface, RootStateInterface> & Actions = {
-  [HousesActionsEnum.SET_HOUSES]: ({ commit }) => {
-    HTTP.get(`/cooperations/1/houses`).then((r) => {
-      commit(HousesMutationsEnum.SET_HOUSES, r.data);
-    });
-  },
+  [HousesActionsEnum.SET_HOUSES]: async ({ commit }) => {
+    const { data } = await HTTP.get(`/cooperations/1/houses`);
```

Comment:
Do you need to hardcode cooperation ID here?

Reply:
It's not my hardcode, but I can correct

Reply:
From the moment these lines are green, this is your hardcode now))

Reply:
I agree) it's my bad